src/ConfigUtil.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Drupal\oe_bootstrap_theme;
+
+use Drupal\Core\Config\FileStorage;
+use Drupal\Component\Utility\Crypt;
+
+/**
+ * Utility class with static methods related to configuration stuff.
+ */
+class ConfigUtil {
+
+  /**
+   * Installs configuration for paragraphs from file.
```

Comment:
Except "oe_bootstrap_theme_paragraphs" there is not hint to paragraphs, we could use this for any config if we jsut remove that bit and send it as a parameter, so lets add here a more generic description.